sdk/eventhubs/azure-messaging-eventhubs-stress/Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM mcr.microsoft.com/java/jdk:11-zulu-alpine as builder
+WORKDIR application
+ARG JAR_FILE=target/*.jar
+COPY ${JAR_FILE} application.jar
+RUN java -Djarmode=layertools -jar application.jar extract
+
+FROM mcr.microsoft.com/java/jdk:11-zulu-alpine
+WORKDIR application
+COPY --from=builder application/dependencies/ ./
+COPY --from=builder application/spring-boot-loader/ ./
+COPY --from=builder application/snapshot-dependencies/ ./
+COPY --from=builder application/application/ ./
+
```

Comment:
It can be useful to have an `ENTRYPOINT` or `CMD` here just so the dockerfile can be run locally for image testing.

Reply:
Sure, I will add the `ENTRYPOINT` for testing purpose.